public/js/components/AtomEmbed/AtomEmbed.js

```diff
@@ -66,26 +70,45 @@ class AtomEmbed extends React.Component {
           <div className="form__row">
             <h3 className="form__subheading">Embed This Atom</h3>
             <div className="form__row">
-              <label htmlFor="embedUrl" className="form__label">Embed URL</label>
-              <input id="embedUrl" className="form__field" value={this.generateEmbedUrl()} readOnly={true}/>
+              <label htmlFor="embedUrl" className="form__label">
+                Embed URL
+              </label>
+              <input
+                id="embedUrl"
+                className="form__field"
+                value={this.generateEmbedUrl()}
+                readOnly={true}
+              />
               <button className="btn" onClick={this.copyUrl}>
-                {this.state.copied ? "Copied!" : "Copy URL"}
+                {this.state.copied ? 'Copied!' : 'Copy URL'}
               </button>
             </div>
           </div>
           <div className="form__row">
             <h3 className="form__subheading">Workflow</h3>
-            <Workflow atom={this.props.atom} config={this.props.config} workflow={this.props.workflow} workflowActions={this.props.workflowActions}/>
+            <Workflow
+              atom={this.props.atom}
+              config={this.props.config}
+              workflow={this.props.workflow}
+              workflowActions={this.props.workflowActions}
+            />
           </div>
           <div className="form__row">
-            <h3 className="form__subheading">Suggest This Atom</h3>
+            <h3 className="form__subheading">
+              Tags
+              {doesAtomTypeRequireTagging(this.props.atom.atomType) && ' *'}
```

Comment:
won't this display `false` if the test is false?

Reply:
Nope! Jsx hides booleans to allow this syntax